submit/views.py

```diff
@@ -1071,6 +1071,10 @@ def submission_view(request, submission, as_user):
     else:
         prev_group = next_group = None
 
+
```

Comment:
This is a bad hack added by P. Conrad for the CS8 Summer 2017 lab exam.

It is ok if it gets reverted.    The purpose was so that students in earlier sections could not share their answers to the lab exam questions with students in later sections.